### PR TITLE
[build] attempt at signing .nupkg

### PR DESCRIPTION
**Pull request by jonathanpeppers**
Based on: https://github.com/xamarin/Xamarin.Android.FSharp.ResourceProvider/pull/8

### CODE REVIEW

**Review by pjcollins — APPROVED** [review]

File: azure-pipelines.yml [hunk 1]
```diff
@@ -47,6 +66,13 @@ jobs:
       artifact: windows
       targetPath: bin
     condition: succeededOrFailed()
+  # NOTE: that PublishBuildArtifacts is required for signing
```
Comment (mattleibow) [inline azure-pipelines.yml:69]: There was a PR to change this, or at least support the other way. I think a property was added. But, this is not required.

Comment (jonathanpeppers) [reply azure-pipelines.yml:69]: It failed when I used `PublishPipelineArtifact` in a weird way:

https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=4392505&view=logs&j=e24aea9e-642d-565f-0ea4-ee7a5a3b8fdc&t=6e777e80-cc0f-5ccb-1c72-bf4d88543f78&l=12

But I'm fine to use `PublishBuildArtifacts`, it works. 👍

Comment (mattleibow) [reply azure-pipelines.yml:69]: Again I don't have a link on my phone, but I found the PR and it was not merged yet. 🙃

Comment (jonathanpeppers) [reply azure-pipelines.yml:69]: This one: https://github.com/xamarin/yaml-templates/pull/96

But I'm fine to leave this as it is.

Comment (mattleibow) [reply azure-pipelines.yml:69]: Yup, but yeah. No need to change. Just was an FYI for future things.

**Pull request merged**